### ISSUE TITLE
Auto BraceCompletion for Generic Members is not triggered on committing an item from Intellisense using `<`.

### ISSUE THREAD

**Issue by balajikris** [dotnet/roslyn#2173]
``` C#
Console.Writ$$
IEnumer$$
```
1. use `(` to commit `Write` from Intellisense.
2. IDE expands it to `Console.Write($$)`
3. use `<` to commit `IEnumerable` from Intellisense.

**_Actual_**:
IDE expands it to `IEnumerable<$$`

**_Expected_**:
IDE expands it to `IEnumerable<$$>`

where `$$` denotes caret position.

Not a regression from VS2013.

**Comment by balajikris** [comment]
I think this is different from #284 because this would work if `IEnumerable` were typed out and then `<` were pressed. However committing `IEnumerable` from Intellisense by pressing `<` won't work. But the actual fix could be related to the fix for #284 as well.

**Comment by balajikris** [comment]
This is another case of #284 . With the identifier to the left of caret incomplete, we have _**no** semantic information_ to determine if we're in a `TypeParameter` or `TypeArgument` context, and `PossibleTypeArgument` returns `false`. Hence, no brace completion here. 

I think this will be hard as none of the solutions suggested at #284 will apply to this directly. We need more than `GetSymbolInfo` here.

**Comment by Pilchie** [comment]
Moving to 1.1 at this point.

**Comment by CyrusNajmabadi** [comment]
I don't repro this:

![notrepro](https://user-images.githubusercontent.com/4564579/76675936-f2604680-657b-11ea-975d-4c5c34931372.gif)
